src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -577,20 +577,23 @@ protected override IEnumerable<Reference> ExtractSelectionResults(Reference sele
      IsVisibleInDynamoLibrary(false)]
     public class DSAnalysisResultSelection : ElementSelection<Element>
     {
+        private const string message = "Select an analysis result.";
+        private const string prefix = "Analysis Results";
+
         public DSAnalysisResultSelection()
             : base(
                 SelectionType.One,
                 SelectionObjectType.None,
-                "Select an analysis result.",
-                "Analysis Results") { }
+                message,
+                prefix) { }
 
```

Comment:
selection nodes contained hardcoded strings for the `message` and `prefix` parameters which have now also been broken out into class constants